src/modules/jcms_article_fragment/jcms_article_fragment.install

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Contains jcms_article_fragment install/update functions.
+ */
+
+/**
+ * Implements hook_update_N().
+ */
+function jcms_article_fragment_update_8001() {
+  \Drupal::entityDefinitionUpdateManager()->applyUpdates();
```

Comment:
The article fragment entity schema has changed in this piece of work from the original piece of work. This simply updates it in places where it's already installed.